statescript/executor.go

```diff
@@ -156,6 +156,8 @@ func execute(name string, timeout time.Duration) int {
 
 	cmd := exec.Command(name)
 
+	stderr, err := cmd.StderrPipe()
```

Comment:
This could be confusing given my last comment, but I think your `if !strings.Contains(name, "Idle") && !strings.Contains(name, "Sync") {` was OK here. I would only change it to `if !strings.HasPrefix(name, "Idle") && !strings.HasPrefix(name, "Sync") {` for safety and performance reasons.